scene/gui/code_edit.h

```diff
@@ -210,14 +223,26 @@ class CodeEdit : public TextEdit {
 
 	void _lines_edited_from(int p_from_line, int p_to_line);
 
+	/* Line length guidelines */
+	TypedArray<int> line_length_guideline_columns;
+	Color line_length_guideline_color;
+
+	/* Symbol lookup */
+	bool symbol_lookup_on_click_enabled = false;
+
+	String symbol_lookup_new_word = "";
+	String symbol_lookup_word = "";
```

Comment:
You don't need to initialize Strings.

Reply:
I missed this, but that can be done as a follow-up (maybe a regex search as there might be more unneeded String inits in the codebase).

Reply:
Yeah there's a ton of them:
```
$ rg 'String .* = "";' | wc -l
106
```